recipe/meta.yaml

```diff
@@ -1,46 +1,45 @@
-{% set version="3.22.0" %}
-{% set ver = version.replace(".", "") + "000" %}
+{% set name = "sqlite" %}
+{% set version = "3.23.1" %}
+{% set ver_major = version.split(".")[0] %}
+{% set ver_minor = version.split(".")[1] %}
+{% set ver_patch = version.split(".")[2] %}
+{% set ver = "".join([ver_major, ver_minor.ljust(3, "0"), ver_patch.ljust(3, "0")]) %}
```

Comment:
Thanks for the improvements here. Maybe we should call this `ver_url` for clarity.